packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.scss

```diff
@@ -10,35 +17,33 @@ $groupHeader-ease-in-back: cubic-bezier(0.600, -0.280, 0.735, 0.045);
   @include focus-border();
 
   &:hover {
-    background: $ms-color-themeLighterAlt;
+    background: $detailsList-item-hover-color;
   }
   &.rootIsSelected {
-    background: $ms-color-themeLighter;
+    background: $detailsList-item-selected-color;
   }
   &.rootIsSelected:hover {
-    background: $ms-color-themeLight;
+    background: $detailsList-item-selected-hover-color;
   }
 }
 
+.focusZone {
```

Comment:
Should we call this focusZone or something more meaningful like say groupHeaderContainer?